spec/ConfigChanges/ConfigChanges.spec.js

```diff
@@ -56,7 +56,7 @@ function innerXML (xmltext) {
     return xmltext.replace(/^<[\w\s\-=\/"\.]+>/, '').replace(/<\/[\w\s\-=\/"\.]+>$/, ''); /* eslint no-useless-escape : 0 */
 }
 
-function get_munge_change (munge, keys) {
+function get_munge_change () {
```

Comment:
This was the only time where some meaningful information might have been lost by enforcing the new linting rule.

However, if one would want to preserve the named arguments, they should simply be used instead of the arguments object. And if that was not an option, there's still [`argsIgnorePattern`](https://eslint.org/docs/rules/no-unused-vars#argsignorepattern).